src/Controls/src/Core/VisualElement/VisualElement.cs

```diff
@@ -906,15 +1075,31 @@ public ResourceDictionary Resources
 			}
 		}
 
+		/// <summary>
+		/// Signals that the platform equivalent element for this element's size has changed and a new layout cycle might be needed.
+		/// </summary>
+		/// <remarks>For internal use only. This API can be changed or removed without notice at any time.</remarks>
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public void PlatformSizeChanged() => InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
 
+		/// <summary>
+		/// Occurs when children of this element are changed in order.
+		/// </summary>
 		public event EventHandler ChildrenReordered;
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/VisualElement.xml" path="//Member[@MemberName='Focus']/Docs/*" />
+		/// <summary>
+		/// Attempts to set focus to this element.
```

Comment:
```suggestion
		/// Attempts to set the keyboard focus on this element.
```
I think we might need to be clearer that this focus is for entries or keyboard-focusable stuff. That being said I'm not even 100% sure what this property does so I might be wrong.